include/lsst/afw/geom/SpanSet.h

```diff
@@ -617,6 +620,23 @@ class SpanSet {
      */
     std::vector<std::shared_ptr<geom::SpanSet>> split() const;
 
+protected:
+    /* Returns the name used by the persistence layer to identify the SpanSet class
+     */
+    virtual std::string getPersistenceName() const;
```

Comment:
Possibly as we don't expect anything to inherit from SpanSets, this is not needed?